include/core/CPersistUtils.h

```diff
@@ -14,17 +14,16 @@
 #include <core/CStringUtils.h>
 #include <core/ImportExport.h>
 
-#include <boost/array.hpp>
 #include <boost/iterator/indirect_iterator.hpp>
 #include <boost/optional.hpp>
-#include <boost/type_traits/integral_constant.hpp>
-#include <boost/type_traits/is_arithmetic.hpp>
-#include <boost/type_traits/remove_const.hpp>
 #include <boost/unordered/unordered_map_fwd.hpp>
 #include <boost/unordered/unordered_set_fwd.hpp>
 
 #include <algorithm>
+#include <array>
+#include <cinttypes>
```

Comment:
`<cstdint>` is more efficient

Reply:
Changed in [fa3ab02](https://github.com/elastic/ml-cpp/pull/1274/commits/fa3ab02793b790aa31efb0c0cd8524a645f0db6a).